### PR TITLE
Search Stackoverflow Extension

### PR DESCRIPTION
**Pull request by sadanand-singh**
## Description

A simple extension to search stackoverflow. Supports addition of tags as well in the search query.

## Type of change

- New extension

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

File: extensions/stackoverflow/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "title": "Search StackOverflow",
+      "subtitle": "StackOverflow",
+      "description": "Search StackOverflow",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.25.0",
+    "node-fetch": "^3.0.0",
+    "raycast": "^1.0.8",
```
Comment (pernielsentikaer) [inline extensions/stackoverflow/package.json:22]: Is this needed?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @sadanand-singh 👋 

It looks like a very useful extension that you're building, nice! 💪 

Found a few things that we need to address before its ready 💯 

- The list icon is not working with dark mode enabled, please look into this
0ooY7kzz6rdE4c8U

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/stackoverflow/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Raycast Search Stackoverflow
+
+This extension brings [stackoverflow](https://stackoverflow.com/) search to raycast!
```
[the remark this answers is not in the document]
Comment (sadanand-singh) [reply extensions/stackoverflow/README.md]: fixed

File: extensions/stackoverflow/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "title": "Search StackOverflow",
+  "description": "Search StackOverflow from Raycast",
+  "icon": "command-icon.png",
+  "author": "reckoning-dev",
+  "version": "1.0.0",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "commandSearchStackOverflow",
+      "title": "Search StackOverflow",
```
[the remark this answers is not in the document]
Comment (sadanand-singh) [reply extensions/stackoverflow/package.json]: fixed

File: extensions/stackoverflow/package.json (hunk 1 above)
Comment (sadanand-singh) [reply extensions/stackoverflow/package.json]: removed

**Review by pernielsentikaer — APPROVED** [review]
Much better 💪 , you could in the next version play with a dark icon and [tint color](https://developers.raycast.com/api-reference/user-interface/icons-and-images#properties-1)

**Pull request merged**